addons/mail/static/src/js/activity.js

```diff
@@ -795,6 +796,9 @@ const ListActivity = KanbanActivity.extend({
             }
             this.$('.o_activity_summary').text(text);
         }
+        if (this.recordData.activity_type_icon) {
+            this.el.querySelector('.o_activity_btn > span').classList.replace('fa-clock-o', this.recordData.activity_type_icon);
+        }
```

Comment:
should be in template with `t-att-class`

Reply:
Please look at [#L704](https://github.com/odoo/odoo/blob/master/addons/mail/static/src/js/activity.js#L704) this is what how classes added to the span tag. If will do in the template then need to use t-jquery inside 'mail.ListActivity' template. Would it be ok?

Reply:
Ok, Let's keep current changes then.

Basically we plan to re-write this file very soon, so it's okay to have some "ugliness" even if it works functionally.